app/components/candidate_interface/application_dashboard_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if @application_form.ended_without_success? %>
+  <%= render CandidateInterface::CarryOverBannerComponent.new(application_form: @application_form) %>
+<% end %>
+
+<h1 class="govuk-heading-xl"><%= title %></h1>
+
+<%= render(CandidateInterface::ApplicationDashboardGuidanceComponent.new(application_form: @application_form)) %>
+<% if @application_form.ended_without_success? %>
+  <%= render CandidateInterface::ApplyAgainCallToActionComponent.new(application_form: @application_form) %>
+<% elsif offers_received_and_all_providers_responded? %>
+  <%= render CandidateInterface::OffersCallToActionComponent.new(application_form: @application_form) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render(CandidateInterface::ApplicationCompleteContentComponent.new(application_form: @application_form)) %>
+    <%= render(CandidateInterface::ApplicationDashboardCourseChoicesComponent.new(application_form: @application_form, editable: false, heading_level: 3, show_status: true, render_link_to_find_when_rejected_on_qualifications: true, display_accepted_application_choices: true)) %>
+    <%= render(CandidateInterface::PreviousApplicationsComponent.new(candidate: @application_form.candidate)) %>
+
+    <h2 class="govuk-heading-m">Updating your contact information</h2>
+    <p class="govuk-body-s">
```

Comment:
Think this should now be the normal size… however when I look at the review app it is using the normal size, so I’m bit confused.  

```suggestion
    <p class="govuk-body">
```